extension.js

```diff
@@ -241,7 +241,7 @@ const TrackerSearchProvider = new Lang.Class({
     },
 
     filterResults : function(results, max) {
```

Comment:
the method filterResults is called by the shell itself: As far as I understand the max is somewhere defined by the shell and should be used. Since I also couldn't find any documentation regarding this, I thought it is safer to use something I've coded - like MAX_RESULTS. (or a simple '5')